README.md

```diff
@@ -219,6 +219,23 @@ animal.set("omnivore", false);
 animal.get("isDirty"); //false
 ```
 
+## Disabling injection
```

Comment:
I still think we should expose ember-cli-simple-store as a service so consumers can `Ember.inject.service`.  The alternative is documenting setting up a new initializer, which not many people are familiar with.

Thoughts?  I can include that in this PR as well and update the README.